ci/terraform/interventions-api-stub/lambda.tf

```diff
@@ -7,7 +7,7 @@ module "account_interventions_stub_role" {
 }
 
 module "account_interventions_stub_lambda" {
```

Comment:
does this need changing?

Reply:
Yup - this new module is a rework of the old stub-endpoint-module (226873160032c33c8708b6d38d173d93af4d1917)